server/routes/prisons/sessionTemplates/editSessionTemplateController.ts

```diff
@@ -30,6 +30,9 @@ export default class EditSessionTemplateController {
         reference,
       )
 
+      const publicClient = sessionTemplate.clients.find(client => client.userType === 'PUBLIC')
+      const showPublicServices = publicClient.active === true ? 'Yes' : 'No'
```

Comment:
Same question about whether `publicClient` here could be `undefined`?

Also `yes | no` vs `Yes | No`.